app/assets/stylesheets/ui/_overlay.scss

```diff
@@ -52,15 +51,28 @@
 
   .overlay-container {
     background: transparent;
+    margin-top: 0;
+    padding-top: 0;
   }
 }
 
+.overlay--fullscreen--green {
+  background-color: rgba(57,163,41, 0.95);
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.